dask/dataframe/core.py

```diff
@@ -3248,9 +3248,34 @@ def pivot_table(self, index=None, columns=None,
         return pivot_table(self, index=index, columns=columns, values=values,
                            aggfunc=aggfunc)
 
-    def to_records(self, index=False):
+    def to_records(self, index=False, lengths=None):
```

Comment:
Reordered arguments following https://github.com/dask/dask/pull/4515#discussion_r265154779

Although index is not currently being used? Should it be added to https://github.com/asmith26/dask/blob/c0cf9f7c1717d7ec40fbda4d6cebbc4421e3d137/dask/dataframe/core.py#L3264 ?